UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs

```diff
@@ -96,18 +96,21 @@ public void SaveGLTFandBin(string path, string fileName)
 			gltfFile.Close();
 			binFile.Close();
 #endif
-
+			GL.sRGBWrite = true;
```

Comment:
Good find! Though really, these two lines should be the only content of this PR. None of the other changes should be included, please strip them.

Reply:
Thanks! I wanted to make a PR ASAP to people can look it at it, but I am planning on making a cleaner one as soon as possible, with only the requested changes included. Expect one within this week :)